scripts/update_specs_from_pricing.py

```diff
@@ -36,6 +36,7 @@
     'Canada (Central)': 'ca-central-1',
     'Asia Pacific (Seoul)': 'ap-northeast-2',
     'AWS GovCloud (US)': 'us-gov-west-1',
+    'AWS GovCloud (US-West)': 'us-gov-west-1',
```

Comment:
Do we need both of these?

Reply:
It fails if I removed `'AWS GovCloud (US)': 'us-gov-west-1',` too.  I was confused too

Reply:
[tested again and looks like it can be removed now](https://github.com/aws-cloudformation/cfn-python-lint/pull/1540)